app/src/androidTest/java/org/dhis2/usescases/event/EventTest.kt

```diff
@@ -105,7 +103,7 @@ class EventTest : BaseTest() {
         }
 
         eventRobot(composeTestRule) {
-            fillRadioButtonForm(radioFormLength)
+//            fillRadioButtonForm(radioFormLength)
```

Comment:
remove comments

Reply:
This is a commented test, but the method does not exists anymore so I left the method commented to have context on the steps when fixing it.